test/Tests/src/Data/Time/Date_Time_Spec.enso

```diff
@@ -190,7 +190,8 @@ spec_with name create_new_datetime parse_datetime nanoseconds_loss_in_precision=
             time . millisecond . should_equal 0
             time . microsecond . should_equal 0
             time . nanosecond . should_equal 0
-            loose_zone_equal time.zone.zone_id  "Etc/UTC"
+            loose_zone_equal (time.zone.zone_id . take (Last 3) . to_case Case.Upper)  "UTC"
+            loose_zone_equal time.zone Time_Zone.utc
```

Comment:
```suggestion
```

Reply:
Fine to remove this line to get CI back working.

For longer term, we will probably want to ensure that this just returns a timezone that `== Time_Zone.utc`.